src/meter/__examples__/stories/LinearMeter.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { Meta, Story } from "@storybook/react";
+
+import { IStyledMeter, StyledMeter as Meter } from "../index";
+
+export default {
+  component: Meter,
+  title: "Component/Meter/Linear",
+} as Meta;
+
+const Base: Story<IStyledMeter> = args => <Meter {...args} />;
+
+export const Playground = Base.bind({});
+Playground.args = {
+  value: 5,
+  min: 0,
+  max: 10,
+  low: 0,
+  high: 10,
+  optimum: 5,
+};
+
+// Examples from https://css-tricks.com/html5-meter-element/
+export const Stories = () => {
+  return (
+    <div>
+      <h4>Case 1 - No attributes</h4>
+      <div>
+        <kbd>{`<Meter />`}</kbd>
+      </div>
+      <br />
+      <Meter />
+      <br />
+      <h4>{`Case 2 - value < max (default: min=0, max=1)`}</h4>
```

Comment:
I think we can clean up these h4 tags, we don't need {``} here

Reply:
I specially need to use them, since `<` cannot be use normally.

Reply:
oh yeah got it!